packages/server/graphql/mutations/helpers/collectReactjis.ts

```diff
@@ -0,0 +1,53 @@
+import Meeting from '../../../database/types/Meeting'
+import {getTeamPromptResponsesByMeetingId} from '../../../postgres/queries/getTeamPromptResponsesByMeetingIds'
+import {DataLoaderWorker} from '../../graphql'
+import isValid from '../../isValid'
+
+const collectReactjis = async (meeting: Meeting, dataLoader: DataLoaderWorker) => {
```

Comment:
I imagine you're intentionally not checking the feature flag here because it's useful to add this data for all meetings so that if we turn the feature flag on, they get to see the Insight sooner rather than later?

I think that's probably the right approach. The downside is it means storing more data in RethinkDB, and we're querying the last 90 days of meetings in `updateTeamInsights` every time we end a meeting.

Reply:
These are valid concerns. I will go with a compromise: collecting the emojis is not that much effort as the data is loaded anyways and we're updating the meeting already. I will change the update method to only run for feature flag owners.